### DISCUSSION TITLE
Change StatusBar color dynamically globally

### DISCUSSION THREAD

**Discussion by maxkoshevoi** [CommunityToolkit/Maui#664] [Q&A]
In XCT doing:
```xml
 <Application.Resources>
        <ResourceDictionary>
            <Style TargetType="Page" ApplyToDerivedTypes="True">
                <Setter Property="xct:StatusBarEffect.Color" Value="{DynamicResource StatusBarColor}" />
                <Setter Property="xct:StatusBarEffect.Style" Value="{DynamicResource StatusBarStyle}" />
            </Style>
...
```

Did the trick. And when I change `StatusBarColor` resource StatusBar color changes automatically thought the application.
But in MCT this code:

```xml
<Setter Property="mct:StatusBarBehavior.StatusBarColor" Value="{DynamicResource StatusBarColor}" />
<Setter Property="mct:StatusBarBehavior.StatusBarStyle" Value="{DynamicResource StatusBarStyle}" />
```
doesn't seem to work.

(full git diff for better context)
![image](https://user-images.githubusercontent.com/6609929/193109045-3db57af6-1962-4d7b-9575-ceb782262d11.png)

**Comment by pictos** [comment]
On XCT those are attached properties, in MCT they're properties tied to the `StatusBarBehavior`. Not sure if `Behaviors` can be used in styles, if so you need to follow the correct pattern (that I don't know what's)

**Comment by CliffAgius** [comment]
Closed and answered
